<?xml version="1.0"?>
<launch>

    <node pkg="ezw_ros_controllers" name="diff_drive_controller" type="diff_drive_controller"
        output="screen">
        <rosparam param="baseline_m">0.485</rosparam>
        <rosparam param="pub_freq_hz">50</rosparam>
	<rosparam param="left_config_file">"/opt/ezw/usr/etc/ezw-smc-core/left_drive_config.ini"</rosparam>
	<rosparam param="right_config_file">"/opt/ezw/usr/etc/ezw-smc-core/right_drive_config.ini"</rosparam>
        <rosparam param="watchdog_receive_ms">10000</rosparam>
    </node>

</launch>
